vizro-core/src/vizro/models/_tooltip.py

```diff
@@ -33,6 +43,16 @@ class Tooltip(VizroBaseModel):
             to see all available arguments. [Not part of the official Vizro schema](../explanation/schema.md) and the
             underlying component may change in the future. Defaults to `{}`.
 
+    Example: `vm.Tooltip` on a `Button`
+        ```python
+        import vizro.models as vm
+
+        vm.Button(
```

Comment:
Not sure whether the button is the clearest example to choose here because it also has `icon` which is maybe confusing with the tooltip icon? Maybe better to do some selector like `vm.RadioItems` instead (which I know you can't see outside a `Filter` without doing `add_type` but that's ok). And to use some `icon` that might make sense there, maybe a warning one?